README.md

```diff
@@ -16,6 +16,8 @@ The app evaluates the readability of page content by comparing the text(foregrou
 - CSS
 - JavaScript
 
+**Please note:** If you are not marked as a contributor for this project, then you will need to follow the first two steps. If you are marked as a contributor, then you can clone the repo `git clone https://github.com/DBenMoshe/contrast-ratio-repo.git` and then follow the rest of the steps in order.
```

Comment:
@jdwilkin4 this text is exactly as was requested in the issue, but I think I need some clarification before merging this PR

`marked as a contributor` - shouldn't we talk about collaborator here, not contributor?

Or 'marked as contributor' and 'collaborator' kinda means the same?

Or I am just super confused by terminology?

Reply:
No, that is a mistake it should be a collaborator. 
@thetalesofj 
If you could make that change, then it should be good to go

Reply:
Got it, thanks for clarification.